src/components/LandingPage/LandingPage.js

```diff
@@ -60,9 +62,11 @@ function LandingPage(props) {
           NuLeaf
         </Typography>
         <Container maxWidth='xs' className={classes.buttonContainer}>
-          <Button variant='contained' color='primary'>
-            Sign up today
-          </Button>
+          <Link to="/register">
+            <Button  variant='contained' color='primary'>
```

Comment:
intead of wrapping Button in Link, ` <Button  variant='contained' color='primary' to='/register' component={RouterLink}>`